api/.pylintrc

```diff
@@ -139,9 +133,8 @@ disable=print-statement,
         deprecated-sys-function,
         exception-escape,
         comprehension-escape,
-
-# Ratinale behind disabled messages:
-
+        missing-class-docstring,
```

Comment:
I'm divided on this. Mostly our classes are models / schemas - which makes having a description redundant. But in general classes need a decent description.

Reply:
I think code reviewer should ask for it then or it should be named.